SPECS/thrift/thrift.spec

```diff
@@ -0,0 +1,691 @@
+ 
+sh ./bootstrap.sh
+ 
+# use unversioned doc dirs where appropriate (via _pkgdocdir macro)
+export PYTHON=%{_bindir}/python3
+%configure --disable-dependency-tracking --disable-static --with-boost=/usr \
+  --docdir=%{?_pkgdocdir}%{!?_pkgdocdir:%{_docdir}/%{name}-%{version}} \
+  %{java_configure} %{ruby_configure} %{erlang_configure} %{golang_configure} %{php_configure} %{lua_configure}
+ 
+# eliminate unused direct shlib dependencies
+sed -i -e 's/ -shared / -Wl,--as-needed\0/g' libtool
+ 
+%make_build
+ 
+%install
+%make_install
+find %{buildroot} -name '*.la' -exec rm -f {} ';'
+find %{buildroot} -name fastbinary.so | xargs -r chmod 755
+find %{buildroot} -name \*.erl -or -name \*.hrl -or -name \*.app | xargs -r chmod 644
+ 
+# Remove javadocs jar
+%if 0%{?want_java} > 0
+find %{buildroot}/%{_javadir} -name lib%{name}-javadoc.jar -exec rm -f '{}' \;
+# Add POM file and depmap
+mkdir -p %{buildroot}%{_mavenpomdir}
+install -pm 644 %{SOURCE1} %{buildroot}%{_mavenpomdir}/JPP-libthrift.pom
+%add_maven_depmap JPP-libthrift.pom libthrift.jar
+%endif
+ 
+# Remove bundled jar files
+find %{buildroot} -name \*.jar -a \! -name \*thrift\* -exec rm -f '{}' \;
+ 
+# Move perl files into appropriate places
+find %{buildroot} -name \*.pod -exec rm -f '{}' \;
+find %{buildroot} -name .packlist -exec rm -f '{}' \;
+
+ 
+%if 0%{?want_php} != 0
+ 
+# Move arch-independent php files into the appropriate place
+mkdir -p %{buildroot}/%{_datadir}/php/
+mv %{buildroot}/%{php_extdir}/Thrift %{buildroot}/%{_datadir}/php/
+%endif
+ 
+# Fix permissions on Thread.h
+find %{buildroot} -name Thread.h -exec chmod a-x '{}' \;
+ 
+# Ensure all python scripts are executable
+find %{buildroot} -name \*.py -exec grep -q /usr/bin/env {} \; -print | xargs -r chmod 755
+%ldconfig_scriptlets
+
+
+%files
+%doc LICENSE NOTICE
+%{_bindir}/thrift
+%{_libdir}/libthrift-%{version}.so
+%{_libdir}/libthriftz-%{version}.so
+%{_libdir}/libthriftnb-%{version}.so
+  
+%files devel
+%{_includedir}/thrift
+%{_libdir}/*.so
+%{_libdir}/*.so.0
+%{_libdir}/*.so.0.0.0
+%exclude %{_libdir}/lib*-%{version}.so
+%{_libdir}/pkgconfig/thrift-z.pc
+%{_libdir}/pkgconfig/thrift-nb.pc
+%{_libdir}/pkgconfig/thrift.pc
+%{_libdir}/pkgconfig/thrift_c_glib.pc
+%doc LICENSE NOTICE
+
+ 
+%if 0%{?want_php} != 0
+%files -n php-%{name}
+%config(noreplace) /etc/php.d/thrift_protocol.ini
+%{_datadir}/php/Thrift/
+%{php_extdir}/thrift_protocol.so
+%doc LICENSE NOTICE
+%endif
+ 
+%if %{?want_erlang} > 0
+%files -n erlang-%{name}
+%{_libdir}/erlang/lib/%{name}-%{version}/
+%doc LICENSE NOTICE
+%endif
+ 
+%files -n python3-%{name}
+%{python3_sitearch}/%{name}
+%{python3_sitearch}/%{name}-%{version}-py%{python3_version}.egg-info
+%doc LICENSE NOTICE
+ 
+%if 0%{?want_java} > 0
+%files -n lib%{name}-javadoc
+%{_javadocdir}/%{name}
+%doc LICENSE NOTICE
+ 
+%files -n lib%{name}-java -f .mfiles
+%doc LICENSE NOTICE
+%endif
+ 
+%changelog
+* Tue Mar 19 2024 Himaja Kesari <himajakesari@microsoft.com>
```

Comment:
I added a comment above to bump the release to 5.
Can you also please add the version information to your changelog entry?
Suggesting: `* Tue Mar 19 2024 Himaja Kesari <himajakesari@microsoft.com> - 0.15.0-5`